examples/js/loaders/GLTFLoader.js

```diff
@@ -261,6 +276,23 @@ THREE.GLTFLoader = ( function () {
 							extensions[ extensionName ] = new GLTFMeshQuantizationExtension();
 							break;
 
+						case EXTENSIONS.EXT_MESHOPT_COMPRESSION:
+							if ( extensionsRequired.includes( extensionName ) ) {
+
+								if ( !this.meshoptDecoder ) {
+
+									throw new Error( 'THREE.GLTFLoader: setMeshoptDecoder must be called before loading compressed files' );	
+
+								}
+
+								if ( !this.meshoptDecoder.supported ) {
+
+									throw new Error( 'THREE.GLTFLoader: MeshoptDecoder support is required to load compressed files' );	
+
+								}
+
+							}
```

Comment:
This doesn't seem like the right place for these checks (this section contains extensions we haven't yet migrated to the new extension API) but I don't immediately know what the right place would be. Does anything break other than less clear user warnings if they're removed? Or we could keep them here and figure out what to do later.

Reply:
This is purely for error clarity - with these removed, you get this:

RangeError: Invalid typed array length: 1048
    at new Float32Array (<anonymous>)
    at GLTFLoader.js:2291

I'm okay with omitting this for now? I also just now realized that BasisU integration suffers from an opposite problem - I've added a similar check to the extension code in my original PR, but this means that if the loader isn't configured, a glTF file with an *optional* extension (and a fallback non-compressed texture) will fail to load if you don't register the KTX loader.

I could omit these error messages from this PR and submit a fix to KTX loader (that will result in poor error handling behavior but it's better than incorrectly failing to load files), and then we could revisit this somehow?

Reply:
I guess this is just something we need to address in the extension API, it seems OK to keep this PR as it is for now. Thanks!